src/news/ArticlePreviewMobile.js

```diff
@@ -0,0 +1,21 @@
+import React, { PropTypes, Component } from 'react';
```

Comment:
Yup the only different between the two is that preview expect an onClick object. I have separated two actually because i thought bringing about a total separation of the two would be better considering the parents components have been separated .